openai/wandb_logger.py

```diff
@@ -13,9 +13,12 @@
     import re
     from pathlib import Path
 
-    from openai import File, FineTune
+    from openai import File, FineTune, FineTuningJob
     from openai.datalib.numpy_helper import numpy as np
-    from openai.datalib.pandas_helper import pandas as pd
+    from openai.datalib.pandas_helper import assert_has_pandas, pandas as pd
+
+
+assert_has_pandas()
```

Comment:
Is this the best place for this? My worry is that we might import from `wandb_logger` upon instantiation of the SDK which will hit this assert even when the WandB stuff isn't being used. Can we localize this assert?